src/front/glsl/mod.rs

```diff
@@ -17,11 +21,19 @@ mod token;
 mod types;
 mod variables;
 
-pub fn parse_str(source: &str, entry: &str, stage: ShaderStage) -> Result<Module, ParseError> {
-    log::debug!("------ GLSL-pomelo ------");
-
+pub fn parse_str(
+    source: &str,
+    entry: &str,
+    stage: ShaderStage,
+    defines: Option<FastHashMap<String, String>>,
```

Comment:
I think the `Option` here is not really needed. If the users don't care, they can always pass `Default::default()`.

Reply:
ok, will fix

Reply:
done